crates/routes/src/images.rs

```diff
@@ -123,15 +123,17 @@ async fn upload(
   let status = res.status();
   let images = res.json::<Images>().await.map_err(error::ErrorBadRequest)?;
   if let Some(image) = &images.files {
-    let uploaded_image = image.first().unwrap();
-    let form = ImageUploadForm {
-      local_user_id: LocalUserId(claims.unwrap().claims.sub),
-      pictrs_alias: uploaded_image.file.to_string(),
-      pictrs_delete_token: uploaded_image.delete_token.to_string(),
-    };
-    ImageUpload::create(&mut context.pool(), &form)
-      .await
-      .map_err(error::ErrorBadRequest)?;
+    if let Some(uploaded_image) = image.first() {
```

Comment:
maybe do this for all images rather than just the first

Reply:
We dont really need to handle upload of multiple images at once. Is there some way to disable that?

Reply:
0.4 and earlier is hardcoded to 10
0.5 makes this value configurable

Reply:
yeah the frontend breaks it up into individual calls with one image, but i suppose other frontends might not do that... Ill see if i can split it up